content/en/synthetics/api_tests/dns_tests.md

```diff
@@ -105,7 +105,11 @@ A notification is sent by your test based on the [alerting conditions](#define-a
     | `{{#is_alert}}`            | Show when the test alerts.                                          |
     | `{{^is_alert}}`            | Show unless the test alerts.                                        |
     | `{{#is_recovery}}`         | Show when the test recovers from alert.                             |
-    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
+    | `{{^is_recovery}}`         | Show unless the test recovers from alert.
+    | `{{#is_renotify}}`         | Show when monitor renotifies   |
+    | `{{^is_renotify}}`         | Show unless monitor renotifies |
+    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)   |
+    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)  |                           |
```

Comment:
Mega-nitpick, though I see it is not aligned on other tables further down this file, so feel free to ignore!
```suggestion
    | `{{^is_recovery}}`         | Show unless the test recovers from alert.                           |
    | `{{#is_renotify}}`         | Show when monitor renotifies                                        |
    | `{{^is_renotify}}`         | Show unless monitor renotifies                                      |
    | `{{#is_priority}}`         | Show when monitor matches priority (P1 to P5)                       |
    | `{{^is_priority}}`         | Show unless monitor matches priority (P1 to P5)                     |
```